app/models/proxy.rb

```diff
@@ -151,8 +151,6 @@ def find_policy_config_by(name:, version:)
   end
 
   def policy_chain
-    # TODO: We need to remove this rolling update as it should be available for everyone using APIcast V2
-    return [] unless provider_can_use?(:policies)
     (policies_config.presence || []).each_with_object([]) do |config, chain|
       chain << config.slice('name', 'version', 'configuration') if config['enabled']
```

Comment:
@hallelujah , these two lines are resulting in a reek error
```
Proxy#policy_chain refers to 'config' more than self (maybe move it to another class?) 
```

What do you think shall we do? I don't see an easy fix except for maybe creating a class `PoliciesConfig` and extract functionality there. But appears to be an overkill on the first sight.

Please let me know how should this be handled.

Reply:
reek is totally right. It could be another PR, or another commit, as you wish. It should be `PolicyChainGenerator` class I think. PolicyConfig does not seem right as the PolicyConfig is per policy and the chain is a chain of policy configs.

Reply:
I'm looking at this.

One slightly related question came though. Do we really need `policies_config.presence || []` here? It seems that we can happily go with `policies_config || []` because we don't mind if `policies_config` is blank or not. Except if we can get an empty string or another unsuitable type here.

Thanks!

Reply:
no because it can be an empty string so presence is mandatory